python/composio/client/enums/enum.py

```diff
@@ -77,6 +77,9 @@ def __hash__(self) -> int:
     def __repr__(self) -> str:
         return f"{self.__class__.__name__}.{self.slug}"
 
+    def __str__(self) -> str:
```

Comment:
Consider adding a docstring to the `__str__` method to maintain consistency with other special methods in the class and improve code documentation:

```python
def __str__(self) -> str:
    """Returns the string representation of the enum value.
    
    Returns:
        str: The slug value of the enum
    """
    return self.slug
```